test/ruby/test_rubyoptions.rb

```diff
@@ -287,6 +287,19 @@ def test_rjit_version
     end
   end
 
+  def test_parser_flag
+    warning = "/Users/hparker/code/ruby/exe/ruby: warning: The Prism compiler is currently experimental and compatibility with parse.y is not yet complete. Please report an issues you find on the prism issue tracker."
```

Comment:
I guess this hardcoded path won't work in CI. It's likely possible to use a regexp or #include? Instead

Reply:
Thanks yeah, updated to not include the path.